spock-core/src/main/java/org/spockframework/runtime/PlatformSpecRunner.java

```diff
@@ -270,8 +264,10 @@ private MethodInfo createMethodInfoForDoRunInitializer(SpockExecutionContext con
     result.setParent(currentFeature.getParent());
     result.setKind(MethodKind.INITIALIZER);
     result.setFeature(currentFeature);
-    for (IMethodInterceptor interceptor : spec.getInitializerInterceptors())
-      result.addInterceptor(interceptor);
+    if (spec.getIsBottomSpec()) {
+      currentFeature.getInitializerInterceptors().forEach(result::addInterceptor);
+    }
+    spec.getInitializerInterceptors().forEach(result::addInterceptor);
```

Comment:
Where would you like that? I thought it was intuitive that the "lower" levels wrap the upper levels. ParentSpec<-ChildSpec<-Feature.

Reply:
Actually from reading the code I first thought the order is different "ParentSpec<-Feature<-ChildSpec".
I was about to ask whether that is intended or it wouldn't make more sense the other way, until I've seen in the test that I misinterpreted the code. :-D

So yes, I agree that the order makes sense.
But imho it would still make sense to document it.
If you think the intuitiveness is enough, that's ok for me too.

As to where, I'm not sure, except within `extensions.adoc` :-D

Reply:
I've added javadoc to the actual methods.